src/Time/TimeSteppers/RungeKutta4.hpp

```diff
@@ -0,0 +1,184 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class RungeKutta4.
+
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <ostream>
+#include <pup.h>
+
+#include "ErrorHandling/Assert.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Time/Time.hpp"
+#include "Time/TimeSteppers/TimeStepper.hpp"  // IWYU pragma: keep
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+struct TimeStepId;
+namespace TimeSteppers {
+template <typename Vars, typename DerivVars>
+class History;
+}  // namespace TimeSteppers
+/// \endcond
+
+namespace TimeSteppers {
+
+/*!
+ * \ingroup TimeSteppersGroup
+ *
+ * The standard 4th-order Runge-Kutta method, given e.g. in
+ * https://en.wikipedia.org/wiki/Runge-Kutta_methods
+ * that solves equations of the form
+ *
+ * \f{eqnarray}{
+ * \frac{du}{dt} & = & RHS(t,u).
+ * \f}
+ * Given a solution \f$u(t^n)=u^n\f$, this stepper computes
+ * \f$u(t^{n+1})=u^{n+1}\f$ using the following equations:
+ *
+ * \f{eqnarray}{
+ * v^{(1)} & = & u^n + dt\cdot RHS(t^n, u^n)/2,\\
+ * v^{(2)} & = & u^n + dt\cdot RHS(t^n + dt/2, v^{(1)})/2,\\
+ * v^{(3)} & = & u^n + dt\cdot RHS(t^n + dt/2, v^{(2)}),\\
+ * v^{(4)} & = & u^n + dt\cdot RHS(t^n + dt, v^{(3)}),\\
+ * v^{(5)} = u^{n+1} & = & (2v^{(1)} + 4v^{(2)} + 2v^{(3)} + v^{(4)} - 3 u^n)/6.
+ * \f}
+ *
+ * Note that in the implementation, the expressio nfor \f$v^{(4)}\f$ is
```

Comment:
Just noticed, there's a character transposition here (` n` -> `n `).  Very minor, so I'm fine with ignoring it.